guide/challenges/branches_arent_just_for_birds.html

```diff
@@ -69,6 +69,8 @@ <h2 class="challenge-name">7 / 11</h2>
       <code>http://githubusername.github.io/repositoryname</code>
 
       <h2>Step: Create a branch</h2>
+      <p>Let's start by creating a new branch on your fork for your contribution:</p>
+      <p><code>$ git checkout gh-pages</code></p>
```

Comment:
OH hai just passing by. This should probably be `$ git checkout -b gh-pages` instead.